irv/irv.py

```diff
@@ -380,11 +378,11 @@ def break_ties(self,
         if self.can_remove_all(tied_candidates, min_nt, tied_val):
             return tied_candidates
 
-        for place in range(self.ballots.shape[1]):
+        for i in range(self.votes[i][1]):
             min_names = []
             min_val = -1
             for name in tied_candidates:
-                place_votes = len(self.ballots[self.ballots[:, place] == name, place])
+                place_votes = len(self.votes[self.votes[:,i] == name, i])
```

Comment:
Nathan is using logical indexing here. Vanilla python lists don’t support this. Here’s an article about it: https://blogs.mathworks.com/loren/2013/02/20/logical-indexing-multiple-conditions/
 (I know this is matlab, but this applies here.)